pageserver/src/http/routes.rs

```diff
@@ -1943,6 +1973,9 @@ pub fn make_router(
         .post("/v1/tenant/:tenant_id/ignore", |r| {
             api_handler(r, tenant_ignore_handler)
         })
+        .post("/v1/tenant/:tenant_shard_id/timeline/:timeline_id/preserve_initdb_archive", |r| {
+            api_handler(r, timeline_preserve_initdb_handler)
```

Comment:
`testing_api_handler` ?

Reply:
This will be used in production so I don't think it is a good idea.